clean/post_ohr.py

```diff
@@ -0,0 +1,52 @@
+import pandas as pd
+import deba
+from lib.clean import float_to_int_str, standardize_desc_cols
+from lib.columns import set_values
+
+
+def drop_missing_agency(df):
+    return df[~((df.agency.fillna("") == ""))]
+
+
+def drop_missing_names(df):
+    return df[~((df.first_name.fillna("") == "") & (df.last_name.fillna("") == ""))]
+
+
+def drop_missing_matched_uid(df):
+    return df[~(df.matched_uid.fillna("") == "")]
+
+
+def uids(df):
+    df.loc[:, "uids"] = df.matched_uid.fillna("").str.cat(
+        df.matched_uid_1.fillna(""), sep=","
+    )
+    df.loc[:, "uids"] = df.uids.str.cat(df.matched_uid_2.fillna(""), sep=",")
+    df.loc[:, "uids"] = df.uids.str.cat(df.matched_uid_3.fillna(""), sep=",")
+    df.loc[:, "uids"] = df.uids.str.cat(df.matched_uid_4.fillna(""), sep=",")
+    df.loc[:, "uids"] = df.uids.str.cat(df.matched_uid_5.fillna(""), sep=",")
+    df.loc[:, "uids"] = df.uids.str.cat(df.matched_uid_6.fillna(""), sep=",")
+    df.loc[:, "uids"] = df.uids.str.cat(df.matched_uid_7.fillna(""), sep=",")
+    df.loc[:, "uids"] = df.uids.str.cat(df.matched_uid_8.fillna(""), sep=",")
+
+    df.loc[:, "uids"] = df.uids.str.replace(r"\,\, ?(.+)?", "", regex=True)
+    return df.fillna("")
+
+
+def clean():
+    df = (
+        pd.read_csv(deba.data("raw/post/post_officer_history.csv"))
```

Comment:
How did you produce this file? Seeing how it has numerous "matched_uid" columns I think it was clearly produced off of existing outputs. It's not a good idea to hide the initial clean, and match code. Once I add the OCR stage to this repo, you should commit the OCR code also.

Reply:
Sounds great. Yes, I have a script for OCR, ner and extraction, clean, match, and fuse.

Reply:
Please include the clean and match code for officer history in this PR.

Reply:
Extracts `officer_name`, `officer_sex`, and the `agency` row which, once cleaned, is split into multiple columns: `agency`, `hire_date`, `left_date`, `left_reason`.